atf_eregs/static/regulations/css/scss/module/custom/_about.scss

```diff
@@ -24,6 +24,19 @@
     background: $atf_carrots;
 }
 
+.about-stopcompare {
+    margin-top: 80px;
+    margin-left: 20px;
+}
+
+.icon-red {
+    color: #d14124;
```

Comment:
It's a shame we're using a hard-coded color in -site here, but thanks for highlighting that. Created https://github.com/eregs/regulations-site/issues/463